### PR TITLE
Igor's branch

### PR DESCRIPTION
**Pull request by IgorTirsina**
Working in us500 TC01

### CODE REVIEW

**Review by VictorIachim — APPROVED** [review]
Please finish as soon as possible

deadline is tomorrow

File: src/test/java/Igor/us500/Us500TC02.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package Igor.us500;
+
+public class Us500TC02 {
+    public static void main(String[] args) {
+        System.out.println("working on the TC 02");
+        System.out.println("still working!");
```
Comment (aliakpa) [inline src/test/java/Igor/us500/Us500TC02.java:6]: how much time do you think this test case will take ?

**Comment by IgorTirsina** [comment]
> Please finish as soon as possible
> 
> deadline is tomorrow

as the devOps comiited their code late, i will try  my best to be on time. thank you

File: src/test/java/Igor/us500/Us500TC02.java (hunk 1 above)
Comment (IgorTirsina) [reply src/test/java/Igor/us500/Us500TC02.java:6]: From my experience, at least 10 hours

**Pull request merged**